Jenkinsfile

```diff
@@ -385,12 +387,18 @@ def nodeWithCleanup(String label, Closure steps) {
   }
 }
 
-def xbuildSafe(String command) {
+def xbuild(String arguments) {
   try {
-    sh "${command}"
+    sh "${xbuildCmd} ${arguments} 1> xbuildOutput"
```

Comment:
I may be wrong but what will happen if sh returns non-0 result? I assume it will throw and stdOut will not be assigned? I need the stdOut in the catch clause as well to check against the GC assertion.